src/Missings.jl

```diff
@@ -207,4 +207,14 @@ missing
 """
 passmissing(f) = PassMissing{Core.Typeof(f)}(f)
 
+
+"""
+    hasmissing(itr) -> Bool
+
+Return `true` if `itr` contains `missing` elements, else `false`.
+
+See also: `ismissing`
+"""
+hasmissing(itr) = eltype(itr) >: Missing && any(ismissing, itr)
```

Comment:
You still need to define a special method for `skipmissing` (as in your original PR I have commented on) for performance reasons:
```
julia> x = Any[i for i in 1:10^6];

julia> y = skipmissing(x);

julia> hasmissing(itr) = eltype(itr) >: Missing && any(ismissing, itr)
hasmissing (generic function with 1 method)

julia> @btime hasmissing($y);
  8.464 ms (0 allocations: 0 bytes)
```

Reply:
True.